src/nets/e2e_asr_attctc_th.py

```diff
@@ -354,6 +355,33 @@ def recognize(self, x, recog_args, char_list, rnnlm=None):
             self.train()
         return y
 
+    def recognize_batch(self, xs, recog_args, char_list, rnnlm=None):
+        prev = self.training
+        self.eval()
+        # subsample frame
+        xs = [xx[::self.subsample[0], :] for xx in xs]
+        ilens = np.fromiter((xx.shape[0] for xx in xs), dtype=np.int64)
+        hs = [to_cuda(self, Variable(torch.from_numpy(xx), volatile=True))
+              for xx in xs]
+
+        # 1. encoder
+        # make a utt list (1) to use the same interface for encoder
+        xpad = pad_list(hs)
+        hpad, hlens = self.enc(xpad, ilens)
+
+        # calculate log P(z_t|X) for CTC scores
+        #if recog_args.ctc_weight > 0.0:
```

Comment:
Shall I remove these comments to use hybrid CTC/Attention? @mn5k

Reply:
Combination of CTC score causes an error at some samples.
So, I recommend you to use the default beam search for hybrid CTC/Attention..

Reply:
@mn5k But the default uses cpu to recognize, right? I'd like to use GPU to accelerate the recognize.

Reply:
Sure. I'll fix it asap.

Reply:
BTW, this behaves differently on pytorch 0.4 and 0.3. When recognizing on pytorch 0.3, it only takes a little memory at around 500MB using batchsize = 1. While it takes 5000MB memory with pytorch 0.4. This is really strange. @mn5k